go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -54,19 +54,21 @@ const (
 	XtraBackup = iota
 	BuiltinBackup
 	Mysqlctld
+	MySQLShell
 	timeout                = time.Duration(60 * time.Second)
 	topoConsistencyTimeout = 20 * time.Second
 )
 
 var (
-	primary       *cluster.Vttablet
-	replica1      *cluster.Vttablet
-	replica2      *cluster.Vttablet
-	replica3      *cluster.Vttablet
-	localCluster  *cluster.LocalProcessCluster
-	newInitDBFile string
-	useXtrabackup bool
-	cell          = cluster.DefaultCell
+	primary          *cluster.Vttablet
+	replica1         *cluster.Vttablet
+	replica2         *cluster.Vttablet
+	replica3         *cluster.Vttablet
+	localCluster     *cluster.LocalProcessCluster
+	newInitDBFile    string
+	useXtrabackup    bool
+	currentSetupType int
```

Comment:
`currentSetupType` looks good. I see it replaces `useXtrabackup` in a few places -- should it completely replace it, such that we can remove `useXtrabackup` altogether?

Reply:
I was a bit unsure about how the removal of the last pieces of `useXtrabackup` would play out, but I did on the last commit and it seems tests are passing locally, but if you could take a better look I would appreciate.